src/utils/getNutrientUnit.ts

```diff
@@ -0,0 +1,17 @@
+export const getNutrientUnit = (key: string) => {
```

Comment:
also, part of the keys we know what to expect, so maybe it could be typed? or maybe we know all of the nutrients?